CONTRIBUTING.md

```diff
@@ -1,145 +1,127 @@
 
-The following guidelines must be followed by all contributors to this repository. Please review them carefully and do not hesitate to ask for help.
+✨ Thank you for taking the time to contribute to this project ✨
 
-### Code of Conduct
+## 📖 Table of Contents
 
-* Review and test your code before submitting a pull request.
-* Be kind and professional. Avoid assumptions; oversights happen.
-* Be clear and concise when documenting code; focus on value.
-* Don't commit commented code to the main repo (stash locally, if needed).
+* [Code of Conduct](#code-of-conduct)
+* [Developing](#developing)
+* [Submitting a new feature](#submitting-a-new-feature)
+* [Reporting bugs](#reporting-bugs)
+* [Contributing](#getting-in-contact)
+* [Coding conventions](#coding-conventions)
 
-### Git Commit Guidelines
+## Code of Conduct
+
+This project adheres to the American Express [Code of Conduct](./CODE_OF_CONDUCT.md). By contributing, you are expected to honor these guidelines.
 
-We follow precise rules for git commit message formatting. These rules make it easier to review commit logs and improve contextual understanding of code changes. This also allows us to auto-generate the CHANGELOG from commit messages.
+## Developing
 
-Each commit message consists of a **header**, **body** and **footer**.
+### Installation
 
-#### Header
+1. Fork the repository `amex-jest-preset` to your GitHub account.
+2. Afterwards run the following commands in your terminal
 
-The header is required and must not exceed 70 characters to ensure it is well-formatted in common git tools. It has a special format that includes a *type*, *scope* and *subject*:
+    ```bash
+    $ git clone https://github.com/<your-github-username>/amex-jest-preset
+    $ cd amex-jest-preset
+    ```
 
-Syntax:
+   > replace `your-github-username` with your github username
 
-```bash
-<type>(<scope>): <subject>
-```
+3. Install the dependencies by running
 
-#### Type
+    ```bash
+    $ cd npm install
+    ```
 
-The *type* should always be lowercase as shown below.
+4. You can now run any of these scripts from the root folder.
 
-##### Allowed `<type>` values:
+#### Running tests
 
-* **feat** (new feature for the user)
-* **fix** (bug fix for the user, not a fix to build scripts)
-* **docs** (changes to documentation)
-* **style** (formatting, missing semi colons, etc; no functional code change)
-* **refactor** (refactoring production code, eg. renaming a variable)
-* **test** (adding missing tests, refactoring tests; no production code change)
-* **chore** (updating build/env/packages, etc; no production code change)
+- `npm run lint`
 
-#### Scope
+Verifies that your code matches the American Express code style defined in
+[`babel-preset-amex`](https://github.com/americanexpress/babel-preset-amex).
 
-The *scope* describes the affected code. The descriptor may be a route, component, feature, utility, etc. It should be one word or camelCased, if needed:
+- `npm pretest`
```

Comment:
I think the bullet points on these commands is funky formatting and hard to read. What about just leaving them as headers w/ no bullet point? Or, if more emphasis is needed, bold them like **`npm pretest`?**

Reply:
thank you for catching this!

Reply:
There's no `pretest` job in this repo.

Reply:
updated on f1a9276